guides/common/modules/proc_assigning-a-rolling-content-view-to-lifecycle-environments.adoc

```diff
@@ -0,0 +1,35 @@
+[id="assigning-a-rolling-content-view-to-lifecycle-environments"]
+= Assigning a rolling content view to lifecycle environments
+
+You can assign your rolling content view to lifecycle environments to limit synchronized content to {SmartProxyServers}.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-assigning-a-rolling-content-view-to-lifecycle-environments[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Lifecycle* > *Content Views*.
+. Select your rolling content view.
+. On the *Details* tab in the *Lifecycle Environments* field, assign your rolling content view to your lifecycle environments.
+
+[id="cli-assigning-a-rolling-content-view-to-lifecycle-environments"]
+.CLI procedure
+. List all content views:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer content-view list \
+--fields id,name \
+--organization "_My_Organization_"
+----
+. Assign your rolling content view to lifecycle environments:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer content-view update \
+--environment-ids _My_List_Of_Lifecycle_Environment_IDs_ \
+--id _My_Rolling_Content_View_ID_
```

Comment:
I believe providing an organization is also required for the `update` command.

Reply:
The parameter is also added to the `create` action. So it is something like

`hammer content-view create --organization-id _My_Organization_ID_ --lifecycle-environments _My_List_Of_Lifecycle_Environment_IDs_   --rolling --name _My Content View Name_

Reply:
Applied.